Makefile

```diff
@@ -10,6 +10,10 @@ dev:
 .PHONY: dev-vagrant
 dev-vagrant:
 	HKNWEB_MODE='dev' pipenv run python ./manage.py runserver [::]:$(DEV_PORT)
+	
```

Comment:
Do you think we could somehow combine all of these `dev-*` using conditional assignment?

Something like

```
IP ?= 127.0.0.1
PORT ?= 3000

.PHONY: dev
dev:
    HKNWEB_MODE='dev' pipenv run python ./manage.py runserver $(IP):$(PORT)
```

and somehow set the Vagrant `IP` env variable to `0.0.0.0`?